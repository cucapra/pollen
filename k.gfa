H	VN:Z:1.0
S	1	CAAATAAG
S	2	A
S	3	G
S	4	T
S	5	C
S	6	TTG
S	7	A
S	8	G
S	9	AAATTTTCTGGAGTTCTAT
S	10	A
S	11	T
S	12	ATAT
S	13	A
S	14	T
S	15	CCAACTCTCTG
P	x	1+,3+,5+,6+,8+,9+,11+,12+,14+,15+	8M,1M,1M,3M,1M,19M,1M,4M,1M,11M
P	y	1+,2+,4+,6+,7+,9+,10+,12+,13+,15+	8M,1M,1M,3M,1M,19M,1M,4M,1M,11M
L	1	+	2	+	8M
L	1	+	3	+	8M
L	2	+	4	+	1M
L	2	+	5	+	0M
L	3	+	4	+	0M
L	3	+	5	+	1M
L	4	+	6	+	1M
L	5	+	6	+	1M
L	6	+	7	+	3M
L	6	+	8	+	3M
L	7	+	9	+	1M
L	8	+	9	+	1M
L	9	+	10	+	19M
L	9	+	11	+	19M
L	10	+	12	+	1M
L	11	+	12	+	1M
L	12	+	13	+	4M
L	12	+	14	+	4M
L	13	+	15	+	1M
L	14	+	15	+	1M
L	15	+	1	+	11M
