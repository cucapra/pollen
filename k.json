{"segment names": ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"], "paths": [0, 2, 4, 5, 7, 8, 10, 11, 13, 14, 0, 1, 3, 5, 6, 8, 9, 11, 12, 14], "path offsets": [0, 10, 20], "path names": ["x", "y"]}