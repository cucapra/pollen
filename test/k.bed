x	0	6	y
