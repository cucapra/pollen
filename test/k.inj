H	VN:Z:1.0
S	1	A
S	2	AAA
S	3	TT
S	4	TT
S	5	GGGG
P	x	1+,2+,3+,4+,5+	*
P	y	2+,3+	*
L	1	+	2	+	0M
L	3	+	4	+	0M
