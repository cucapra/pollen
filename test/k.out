#node.id	depth	depth.uniq
1	2	2
2	1	1
3	1	1
4	1	1
5	1	1
6	2	2
7	1	1
8	1	1
9	2	2
10	1	1
11	1	1
12	2	2
13	1	1
14	1	1
15	2	2
